Dockerfile

```
@@ -17,7 +17,7 @@ RUN yarn verify
 RUN yarn build
 
 # Host project in nginx
-FROM nginx
+FROM nginx:1.16.1-alpine
```

What do you think about using 1.16?

```suggestion
FROM nginx:1.16-alpine
```

In this way, if there as new patch version it will be updated.